rotkehlchen/tests/unit/test_cost_basis.py

```diff
@@ -1111,66 +1119,49 @@ def test_taxable_acquisition(accountant: Accountant) -> None:
 @pytest.mark.parametrize(('db_settings', 'expected_pnls'), [
     (
         {'cost_basis_method': CostBasisMethod.FIFO, 'include_fees_in_cost_basis': False},
-        [ZERO, ZERO, FVal(-10), FVal(3500), ZERO, FVal(-10), ZERO, ZERO, FVal(-10), FVal(1600), ZERO, FVal(-10)],  # noqa: E501
+        [FVal(0), FVal(-10), FVal(0), FVal(3500), FVal(0), FVal(-10), FVal(0), FVal(-10), FVal(0), FVal(1600), FVal(0), FVal(-10)],  # noqa: E501
```

Comment:
Why not use `ZERO` as before? Please do. Also here it's just the order that change due to a function order change you did?